packages/jest-resolve/package.json

```diff
@@ -11,6 +11,7 @@
     "browser-resolve": "^1.11.2",
     "jest-file-exists": "^17.0.0",
     "jest-haste-map": "^18.1.0",
+    "jest-util": "^18.1.0",
```

Comment:
I did that because `getModulePath` is used in `jest-runtime` too and I didn't want to have code repetition. I will move it to be a static method in the `jest-resolve`.

Is it fine to keep the `getModulePath` in `jest-utils` and use it in `jest-runtime`? Or should I move it to a static method in `jest-runtime`?